workflows/validate-members.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+URLS=$(git diff origin/main..HEAD --unified=0 -- ./data/members.json  | grep '^+' | grep -o '\"url\":\s\".*\"' | grep -Eo "(http|https)://[a-zA-Z0-9./?=_%:-]*" | sort -u)
+# Use a for loop to iterate over the URLs in the $URLS variable
+# Check if URLs is not empty if so exit 0
+if [ -z "$URLS" ]; then
+	exit 0
+fi
+
+errors=0
+for url in $URLS; do
+  # Use curl and grep to search for the <the-claw-webring-widget> tag in the URL
+  curl -s $url | grep -qo "<the-claw-webring-widget>"
```

Comment:
I know that some people are including the widget, but it's NOT on their home page. I wonder how to get round this. Maybe we ask that people use a URL in members.json to the page that the widget is on?

Reply:
Yeah, I think that might be best if it's not on the page where the webring is surely it breaks down then as you break the chain and are unable to quickly cycle to the next person in the chain

Reply:
My 2 cents. While I agree it breaks the chain. 

For me. The widget is just too big and different to the rest of my site to have it on the homepage. 

I have a link to the webring page in my left sidebar. Ideally, I would love just a next or previous links I can add into the sidebar making it always accessible and available. 

Maybe a v2 for the component?

Reply:
Perhaps validation/authentication can be similar to how webmentions do it with a link in the header ?

Reply:
> My 2 cents. While I agree it breaks the chain.
> 
> For me. The widget is just too big and different to the rest of my site to have it on the homepage.
> 
> I have a link to the webring page in my left sidebar. Ideally, I would love just a next or previous links I can add into the sidebar making it always accessible and available.
> 
> Maybe a v2 for the component?

But in that situation would you not put the webring url in as the `url:` within `members.json`? 

Users are still landing on your site like a landing page, so wouldn't it be best to introduce people to your site from a webring page?

Reply:
> > My 2 cents. While I agree it breaks the chain.
> > For me. The widget is just too big and different to the rest of my site to have it on the homepage.
> > I have a link to the webring page in my left sidebar. Ideally, I would love just a next or previous links I can add into the sidebar making it always accessible and available.
> > Maybe a v2 for the component?
> 
> But in that situation would you not put the webring url in as the `url:` within `members.json`?
> 
> Users are still landing on your site like a landing page, so wouldn't it be best to introduce people to your site from a webring page?

Perhaps, but normally your homepage tells people who you are and what you do, so this is duplicating content, past saying I’m a member of the claw webring what are you saying that’s different to what is already said on the homepage…?

Reply:
> > > My 2 cents. While I agree it breaks the chain.
> > > For me. The widget is just too big and different to the rest of my site to have it on the homepage.
> > > I have a link to the webring page in my left sidebar. Ideally, I would love just a next or previous links I can add into the sidebar making it always accessible and available.
> > > Maybe a v2 for the component?
> > 
> > 
> > But in that situation would you not put the webring url in as the `url:` within `members.json`?
> > Users are still landing on your site like a landing page, so wouldn't it be best to introduce people to your site from a webring page?
> 
> Perhaps, but normally your homepage tells people who you are and what you do, so this is duplicating content, past saying I’m a member of the claw webring what are you saying that’s different to what is already said on the homepage…?

I suppose this would be a wider discussion (perhaps another issue?)  but my feelings on it, are that the webring should be on the page you land on, to allow the user to continue navigating through the other members with ease.

Not having on the page you land then means the user has friction in the form that they then need to go and manually find the site's webring page and click next to continue, and that friction will surely turn off a large number of users and defeats the idea of the webring?

Again I think a wider discussion is needed on this outside of this issue, as I think whatever path is chosen there will be drawbacks.